image/image.go

```diff
@@ -176,25 +178,23 @@ func Prepare(opts *Options) error {
 		return err
 	}
 
+	if model.Architecture() != "" && opts.Architecture != "" && model.Architecture() != opts.Architecture {
+		return fmt.Errorf("cannot override model architecture: %s", model.Architecture())
```

Comment:
I suspect we need to go over all these with `i18n.G` (not in this PR though)

Reply:
yes, no i18n at all in image, but given the usage it should probably have it. But for a different PR/somebody else to do.